### ISSUE TITLE
Modeldoc - Rotate Modifier

### ISSUE THREAD

**Issue by gvarados1** [Facepunch/sbox-issues#2330]
### For?

Other

### What can't you do?

Modeldoc has Translate and Scale&Mirror modifiers but no Rotate modifier

### How would you like it to work?

There should be a global rotate modifier similar to rotate and scale&mirror
![image](https://user-images.githubusercontent.com/39461072/190879614-11d29d5e-f828-4e20-92af-4542d56f4275.png)

### What have you tried?

Currently we have to change the rotation of every single mesh. I can be very annoying when you have 4 lods and a physics mesh.

**Comment by DoctorGurke** [comment]
I'd also suggest a complete transform node, that includes translate, rotate and scale which applies to everything

**Comment by gvarados1** [comment]
Yeah it makes sense to have all this in one node but if the originals were removed it'd be a maaasssiive pain day

**Comment by DoctorGurke** [comment]
compiled models would still work after nodes are removed/changed, fixes would only need to be applied when you want to edit a model anyways
